### PR TITLE
Create README-kr.md

### PR DESCRIPTION
**Pull request by hyojeong-creative**
update for Korean verision of README

### CODE REVIEW

File: README-kr.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+$ fvm flutter run -d ${deviceのid} # 이와 같은 경우、93E3B70F-B378-4295-BC3E-5DAA4958D463
+```
+
+## 추천 개발 환경
+* AndroidStudio　https://developer.android.com/studio
+```
+Android Studio로 작업하는 경우 예시
+
+Preferences > Language & Frameworks > 아래와 같이 Flutter SDK 경로로 접근하세요.
+/{프로젝트로 접근하기}/.fvm/flutter_sdk
```
Comment (Fleta) [inline README-kr.md:44]:
```suggestion
/{프로젝트의 루트 디렉토리}/.fvm/flutter_sdk
```

I'm leaving suggestion it seems slightly different. I guess you could say it's the root directory.

**Review by kur00 — APPROVED** [review]
LGTM!!

Thank you for your translating!

**Review by Fleta — COMMENTED** [review]
I'm leaving suggestion!

File: README-kr.md (hunk 1 above)
Comment (Fleta) [reply README-kr.md:44]: oh it's already merged.

Comment (ayuki-joto) [reply README-kr.md:44]: @Fleta 
Please make a PR for this correction!

Comment (Fleta) [reply README-kr.md:44]: @ayuki-joto I'll make a fixing PR later. (Maybe today night?) Thanks for replying.

Comment (ayuki-joto) [reply README-kr.md:44]: I am looking forward to it!
If you need any help please contact me!

**Pull request merged**